docs/components/demo/Autocomplete/css/styles.css

```
@@ -0,0 +1,114 @@
+  color: var(--grass-11);
+}
+
+.AutocompleteContent {
+  z-index: 10;
+  width: 100%;
+  position: absolute;
+  overflow: hidden;
+  background-color: white;
+  border-radius: 6px;
+  margin-top: 8px;
+  box-shadow: 0px 10px 38px -10px rgba(22, 23, 24, 0.35), 0px 10px 20px -15px rgba(22, 23, 24, 0.2);
```

Stylelint (17.2.0)

[error] Expected "0.35" to be "35%" (alpha-value-notation)

(alpha-value-notation)